### DISCUSSION TITLE
Macro "if" functions dont work.

### DISCUSSION THREAD

**Discussion by Funky-Punky** [sezanzeb/input-remapper#188] [Q&A] [answered]
As the Title says: when i try to use an "if_tap", "if_single" or "if_eq" macro function the debug information in the bottom left reads: "Exception: unknown function if_tap". I also tried copying some of the examples from your example page, but that showed the same result.
Am i dumb or is this broken/not a feature any more?

**Comment by sezanzeb** [comment, marked as answer]
How did you install key-mapper? Please post the output `key-mapper-control --version`

Those features are really new and not yet in the release .deb files

**Reply by Funky-Punky** [reply]
Oh well then that's it. I used the .deb file to install it. Thanks for the quick answer!

**Reply by Funky-Punky** [reply]
just reinstalled via ait and command line - it works now
